src/client/pages/ToeristischeVerhuurDetail/VergunningVerhuur.tsx

```diff
@@ -68,7 +72,10 @@ function useStatusLineItems(
     if (vergunning.caseType === 'B&B - vergunning') {
       lineItems.push({
         id: 'item-3',
-        status: vergunning.decision || '',
+        status:
+          vergunning.status === 'Afgehandeld'
+            ? vergunning.decision || ''
```

Comment:
Maak hier ook maar ipv `''` afgehandeld van. We liegen er niet mee!

Reply:
Of natuurlijk: `vergunning.status === 'Afgehandeld' && vergunning.decision ? vergunning.decision: 'Afgehandeld'`